logviewer/view_log_progress.cgi

```diff
@@ -0,0 +1,47 @@
+#!/usr/local/bin/perl
+# view_log_progress.cgi
+# Returns progressive output for some system log
+
+require './logviewer-lib.pl';
+&ReadParse();
+&foreign_require("proc", "proc-lib.pl");
+
+# System log to follow
+my @systemctl_cmds = &get_systemctl_cmds(1);
+my ($log) = grep { $_->{'id'} eq $in{'idx'} } @systemctl_cmds;
```

Comment:
What if there is no matching log?

Reply:
It will fail on lines 19-23.

Reply:
You should add a check immediately after line 11 for $log being set

Reply:
Right, fixed here https://github.com/webmin/webmin/commit/dccc3fb10e3f9dd85f0f14be1e2666bf183d655f.